install_sct

```diff
@@ -599,13 +599,11 @@ python/bin/conda create -y -n venv_sct python=3.7
 find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +
 
 # make sure that there is no conflict with local python install by making venv_sct an isolated environment.
-# workaround for https://github.com/conda/conda/issues/7173
+# thanks @javidcf https://github.com/conda/conda/issues/7173#issuecomment-884098367
 # see also
 # * https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3067
 # * https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3200
-# this needs to be added very early in python's boot process
-# so using sitecustomize.py or even just appending to the file are impossible.
-perl -pi -e 's/^ENABLE_USER_SITE.*$/ENABLE_USER_SITE = False/' "$SCT_DIR/$PYTHON_DIR/envs/venv_sct/lib/python"*"/site.py"
+python/bin/conda env config vars set PYTHONNOUSERSITE=1 -n venv_sct
```

Comment:
Nice find!! :heart: 

I added a fix in caaf5b2. We have to specify `python/bin/conda` here because `source conda.sh` doesn't get run until a few lines later:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/caaf5b200e71db431a36d44f3afcf2d92d68cfd6/install_sct#L610

Alternatively, we could move the `source` line up a bit, so that we can use `conda` direcly.

```diff
# Run conda installer
print info "Installing Miniconda..."
run bash "$TMP_DIR/miniconda.sh" -p "$SCT_DIR/$PYTHON_DIR" -b -f
+# shellcheck disable=SC1091
+source python/etc/profile.d/conda.sh

# Workaround for WSL "HTTP 000 Connection Failed" error
# See: https://github.com/conda/conda/issues/9948#issuecomment-909989810
# Syntax explanation: https://stackoverflow.com/a/6085237
find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +

# create conda environment
print info "Creating conda environment..."
-python/bin/conda create -y -n venv_sct python=3.7
+conda create -y -n venv_sct python=3.7

# Reapply the touch fix to avoid `pip` connection issues on WSL
find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +

# make sure that there is no conflict with local python install by making venv_sct an isolated environment.
# thanks @javidcf https://github.com/conda/conda/issues/7173#issuecomment-884098367
# see also
# * https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3067
# * https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3200
-python/bin/conda env config vars set PYTHONNOUSERSITE=1 -n venv_sct
+conda env config vars set PYTHONNOUSERSITE=1 -n venv_sct

# activate miniconda
-# shellcheck disable=SC1091
-source python/etc/profile.d/conda.sh
set +u #disable safeties, for conda is not written to their standard.
conda activate venv_sct
set -u # reactivate safeties
```

Reply:
Alternatively, we could move the `source` line up a bit, so that we can use `conda` direcly.

```diff
# Run conda installer
print info "Installing Miniconda..."
run bash "$TMP_DIR/miniconda.sh" -p "$SCT_DIR/$PYTHON_DIR" -b -f
+# shellcheck disable=SC1091
+source python/etc/profile.d/conda.sh

# Workaround for WSL "HTTP 000 Connection Failed" error
# See: https://github.com/conda/conda/issues/9948#issuecomment-909989810
# Syntax explanation: https://stackoverflow.com/a/6085237
find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +

# create conda environment
print info "Creating conda environment..."
-python/bin/conda create -y -n venv_sct python=3.7
+conda create -y -n venv_sct python=3.7

# Reapply the touch fix to avoid `pip` connection issues on WSL
find "$SCT_DIR/$PYTHON_DIR" -type f -exec touch {} +

# make sure that there is no conflict with local python install by making venv_sct an isolated environment.
# thanks @javidcf https://github.com/conda/conda/issues/7173#issuecomment-884098367
# see also
# * https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3067
# * https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3200
-python/bin/conda env config vars set PYTHONNOUSERSITE=1 -n venv_sct
+conda env config vars set PYTHONNOUSERSITE=1 -n venv_sct

# activate miniconda
-# shellcheck disable=SC1091
-source python/etc/profile.d/conda.sh
set +u #disable safeties, for conda is not written to their standard.
conda activate venv_sct
set -u # reactivate safeties
```